githubbot/main.go

```diff
@@ -53,26 +53,21 @@ func (s *BotServer) makeAdvertisement() kbchat.Advertisement {
 	subExtended := fmt.Sprintf(`Enables posting updates from the provided GitHub repository to this conversation.
 
 Example:%s
-!github subscribe keybase/client%s`,
-		backs, backs)
+!github subscribe keybase/client%s
+
+Subscribe to a specific branch:%s
+!github subscribe facebook/react gh-pages%s`,
```

Comment:
The `Title` of these commands should be updated to have the optional branch arg shown, like `*!github subscribe* <username/repo> [branch]`